recipes/spherely/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "spherely" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/benbovy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 84d0634b6e22bfffc4d6730c70a7f4bfaa7a1ca05e57aa6fd83cd71421247b23
+
+build:
+  number: 0
+  skip: true  # [py<310 or python_impl == 'pypy']
+  script: python -m pip install . -vv
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make  # [linux]
+  host:
+    - python
+    - pip
+    - scikit-build-core
+    - pybind11 >=2.10.0
+    - s2geometry >=0.11.1
+    - s2geography >=0.1.2
```

The libraries should get a `run_export` and added for conda-forge-pinning. Otherwise, this will not lead to a sustainable pinning.
```suggestion
    - s2geometry
    - s2geography
```

---

For s2geograhy, we are the first package that depends on it, is that worth to use a global pinning for?

---

Yes, because this will make sure the rebuilds happen via the bot.

---

I was considering doing that afterwards for s2geometry anyway, since at least one other package (pys2index) depends on it.

---

But that requires you to manually start a migration? (or is starting the rebuilds also already automated?)

My main concern is the overhead in managing this on the short term, with s2geography and s2geometry being mostly developed by the same people, and so we can easily do PRs in the respective feedstocks when needed.

---

The bot should open the respective PRs automatically. It will add 2-3h of delay compared to manually doing it but it ensures that the pinnings are always compatible.

---

> The bot should open the respective PRs automatically. It will add 2-3h of delay compared to manually doing it but it ensures that the pinnings are always compatible.

Sounds good!